aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -119,6 +123,7 @@ public void configure(
     ) throws AuthException {
         final CountDownLatch latch = new CountDownLatch(1);
         final AtomicReference<Exception> asyncException = new AtomicReference<>();
+        LogFactory.setLevel(LogFactory.Level.OFF);
```

Comment:
I do wonder idly if we could do a kludge a thing where we listen to the logs and reply them all as `DEBUG`.  A bad idea for later maybe.

Reply:
Maybe - I kinda want to get away from logging stuff in favor of Hub and Callbacks but we should discuss as a team what we want our policy to be on that,